app/src/main/java/io/github/wax911/retgraph/model/Repository.kt

```diff
@@ -1,7 +1,10 @@
 package io.github.wax911.retgraph.model
 
+import kotlinx.serialization.Serializable
+
+@Serializable
 data class Repository(
-        val full_name: String?,
-        val name: String?,
-        val owner: User?,
+        val full_name: String? = null,
```

Comment:
Kotlinx Serialization fails on runtime if it does not receive any `full_name` in the response, so we have to set a default value for anything that might not be part of the response. For graphql I suppose it could be anything (except ID/slug perhaps).